src/libs/actions/IOU.js

```diff
@@ -221,11 +221,15 @@ function buildOnyxDataForMoneyRequest(
                           [chatCreatedAction.reportActionID]: {
                               errors: ErrorUtils.getMicroSecondOnyxError('iou.error.genericCreateFailureMessage'),
                           },
+                          [reportPreviewAction.reportActionID]: {
+                              errors: ErrorUtils.getMicroSecondOnyxError(null),
+                          },
```

Comment:
There was no error before, we need this for the opacity effect.

I've only added this error if it's a new chat report and not for existing chat reports. The reason for this is because this particular report action is re-used whenever an IOU is made.